sdk/src/builder.rs

```diff
@@ -1380,7 +1395,7 @@ impl Builder {
                 crate::utils::thumbnail::make_thumbnail_bytes_from_stream(
                     format,
                     &mut stream,
-                    settings,
+                    self.context.settings(),
```

Comment:
I wonder if this is an API where we could benefit from passing in the full context too.